react/src/components/Widget/Widget.tsx

```diff
@@ -148,7 +148,7 @@ export const Widget: React.FC<WidgetProps> = props => {
   const price = props.price;
   const [url, setUrl] = useState('');
   const [userEditedAmount, setUserEditedAmount] = useState<currencyObject>();
-  const [text, setText] = useState('Send any amount of BCH');
+  const [text, setText] = useState('Send any amount of XEC');
```

Comment:
Here and in the other places you changed hardcoded `BCH`  to `XEC` we could have `defaultStorybookCurrency?.toUpperCase()`.

Reply:
> Here and in the other places you changed hardcoded `BCH` to `XEC` we could have `defaultStorybookCurrency?.toUpperCase()`.

sure, I will make sure to replace this with a logic to reference the default currency name